plugins/webp-uploads/hooks.php

```diff
@@ -362,7 +385,7 @@ function webp_uploads_filter_image_editor_output_format( $output_format, ?string
 function webp_uploads_remove_sources_files( int $attachment_id ): void {
 	$file = get_attached_file( $attachment_id );
 
-	if ( empty( $file ) ) {
+	if ( false === (bool) $file ) {
```

Comment:
Since the function returns `string|false` I think the type cast can be removed:

```suggestion
	if ( false === $file ) {
```

Reply:
That’s a fair point! The reason I added the type cast was because the return value of `get_attached_file()` is filterable through the `get_attached_file` filter. This means it’s possible for the `$file` value to be modified into an empty string, and the cast was a way to account for that edge case.

If you think handling this specific scenario isn’t necessary, I can remove the type cast since it’s not strictly required otherwise. Let me know what you think!

Reply:
Ah! Good point yourself! Nevermind then.